### ISSUE TITLE
Have the MSE included in predict CLI command output

### ISSUE THREAD

**Issue by regel** [regel/loudml#25]
1.3.0

We save the MSE (validation loss) to disk when training a model. This information should also be included in the JSON status returned in the predict CLI command.

This would allow users to find if there is overfit, or not, by comparing the 2 MSE values and evaluate the model behaviour over time. Or, after training and using the model decide if it is necessary to re-train the model on new data.

**Comment by vnyb** [comment]
MSE has been added to `on_anomaly_start()` on anomaly detection. It is also available when running CLI with `--anomalies` option on `predict`.

**Comment by regel** [comment]
Title changed. In fact it's the loss value that would be good to have when using loudml predict CLI, in order to be able to compare it with the training loss value.
